cl/phase1/network/services/constants.go

```diff
@@ -29,8 +29,8 @@ const (
 	blobJobsIntervalTick          = 5 * time.Millisecond
 	singleAttestationIntervalTick = 10 * time.Millisecond
 	attestationJobsIntervalTick   = 100 * time.Millisecond
-	blockJobExpiry                = 7 * time.Minute
-	blobJobExpiry                 = 7 * time.Minute
+	blockJobExpiry                = 24 * time.Second
```

Comment:
I am not sure but just curious, is this expiry time too short?
Does that mean we have only 24 secs to catch up block?

Reply:
It means that if we get a block from gossip and we dont have its parent, we memorize for 24 seconds, which would be 2 slots approximatelly. 7 minutes lead to high memory use, especially for blobs